### ISSUE TITLE
Experimental setup for comparison with baseline

### ISSUE THREAD

**Issue by Flyingkeyboard699** [ruizhang-ai/L0-SIGN_Detecting_Benefitial_Feature_Interactions_for_Recommender_Systems#4]
This is an interesting work. When I realized the baseline of Frappe data set, I found that the evaluation metrics were better than those reported in this paper. I want to know whether your baseline hyperparameter setting is consistent with the model proposed in this paper or whether grid search is the best.

**Comment by ethanmock** [comment]
Hi,

We use the DeepCTR package (https://deepctr-doc.readthedocs.io/en/latest/index.html) to run the baseline models. The MLP structure and other settings are consistent with our model for a fair comparison.

**Comment by Flyingkeyboard699** [comment]
Hi，ethanmock.
Thanks for the reply. I found that the learning rate and regularization rate have a great impact on the final result. I want to know whether the two hyper-parameters settings of the baseline model refer to DeepCTR package or are consistent with those of your model.
